{"name": "test_login_02[15200000001-1111111]", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ntest\\test_loginRegister.py:27: AssumptionFailure\n>>\tpytest.assume(msg == '密码错误')\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               test\\test_loginRegister.py:27: AssumptionFailure\nE               >>\tpytest.assume(msg == '密码错误')\nE               AssertionError: assert False\n\nD:\\Program Files (x86)\\Python39\\lib\\site-packages\\six.py:702: FailedAssumption"}, "parameters": [{"name": "mobile", "value": "'15200000001'"}, {"name": "password", "value": "'1111111'"}], "start": 1626751751241, "stop": 1626751751240, "uuid": "f75b08f6-7093-4582-ab3e-a651add9aacf", "historyId": "664a61fbf50a833c317512019294682e", "testCaseId": "5656a2d4152952ad4657e85d32171417", "fullName": "test.test_loginRegister.TestLoginRegister#test_login_02", "labels": [{"name": "story", "value": "用错误的密码登录"}, {"name": "feature", "value": "测试注册登录接口"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_loginRegister"}, {"name": "subSuite", "value": "TestLoginRegister"}, {"name": "host", "value": "LAPTOP-5J4VKD8M"}, {"name": "thread", "value": "7532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_loginRegister"}]}